.github/workflows/graphql.yml

```
@@ -0,0 +1,42 @@
+name: GraphQL API Documentation
+on:
+  push:
+    branches:
+      - 'master'
+    paths:
+      - 'internal/graphql/schema/schema.graphql'
+  pull_request:
+    branches:
+      - 'master'
+    paths:
+      - 'internal/graphql/schema/schema.graphql'
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    if: github.repository == 'meshery/meshery'
+    steps:
+    - uses: actions/checkout@v2
+      with:
+          token: ${{ secrets.GH_ACCESS_TOKEN }}
+          ref: 'master'
+
+    - name: Set up Ruby
+      uses: ruby/setup-ruby@473e4d8fe5dd94ee328fdfca9f8c9c7afc9dae5e

```

@amino19 any reason for using this particular commit and not a stable release of the setup-ruby action?

---

@DelusionalOptimist : Are you concern about <a href= "https://docs.github.com/en/actions/guides/building-and-testing-ruby"> this</a>? Is it standard?

---

@DelusionalOptimist : Actually doesn't have any specific reason, just I saw this in GitHub marketplace while started writing workflow. :slightly_smiling_face: 
BTW, Are you concern about <a href= "https://docs.github.com/en/actions/guides/building-and-testing-ruby"> this</a>? Is it standard?

---

```suggestion
      uses: ruby/setup-ruby@v1
```
I see. So let's use the v1 release to maintain consistency with other workflows. :+1: